### ISSUE TITLE
Project fails to build

### ISSUE THREAD

**Issue by valarauca** [spring-projects/spring-petclinic#305]
So I followed the instructions on the readme

```
	git clone https://github.com/spring-projects/spring-petclinic.git
	cd spring-petclinic
	./mvnw spring-boot:run
```

But the last stage doesn't start instead I see

```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.webjars:webjars-locator:jar is missing. @ line 85, column 17
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project org.springframework.samples:spring-petclinic:2.0.0 (/home/cody/Documents/java_stuff/spring-petclinic/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.webjars:webjars-locator:jar is missing. @ line 85, column 17
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
```

Not sure what is happening?

**Comment by arey** [comment]
Hi @valarauca, the org.webjars:webjars-locator artefact seems to have changed its gav. Try to rename the webjars-locator artefactId to webjars-locator-core. It's working on my desktop.

**Comment by arey** [comment]
@snicoll fixes the code by this commit: cdd802eeee36f8083580a6cd0368b007cb4a0bad
